client/settings/payment-methods/index.js

```diff
@@ -22,24 +23,31 @@ const IconsWrapper = styled.ul`
 	}
 `;
 
-const PaymentMethodsDescription = () => (
-	<>
-		<h2>
-			{ __(
-				'Payments accepted on checkout',
-				'woocommerce-gateway-stripe'
-			) }
-		</h2>
-		<p>
-			{ __(
-				'Add and edit payments available to customers at checkout. ' +
-					'Based on their device type, location, your customers will ' +
-					'only see the most relevant payment methods.',
-				'woocommerce-gateway-stripe'
+const PaymentMethodsDescription = () => {
+	const { isUpeEnabled } = useContext( UpeToggleContext );
+
+	return (
+		<>
+			<h2>
+				{ __(
+					'Payments accepted on checkout',
+					'woocommerce-gateway-stripe'
+				) }
+			</h2>
+
+			{ isUpeEnabled && (
```

Comment:
In the designs the description is not present when UPE is disabled - fixed here too.